proto/tendermint/types/evidence.proto

```diff
@@ -28,7 +28,7 @@ message DuplicateVoteEvidence {
 message LightClientAttackEvidence {
   tendermint.types.LightBlock         conflicting_block = 1;
   int64                               common_height     = 2;
-  repeated tendermint.types.Validator byzantine_validators = 3 [(gogoproto.nullable) = false];
+  repeated tendermint.types.Validator byzantine_validators = 3;
```

Comment:
why remove nullable? can byzantine validators be empty?

Reply:
In the case of amnesia evidence yes